CHANGELOG.md

```diff
@@ -94,6 +94,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#2097](https://github.com/nix-rust/nix/pull/2097))
 - Add the ability to set `kevent_flags` on `SigEvent`.
   ([#1731](https://github.com/nix-rust/nix/pull/1731))
+- Added `F_GETPATH` FcntlFlags entry on Apple/NetBSD/DragonflyBSD for `::nix::fcntl`.
```

Comment:
This seems to be in the wrong place, it should be put under:

```
## [Unreleased] - ReleaseDate
### Added
```

Reply:
ah true..